src/constants.js

```diff
@@ -59,6 +59,9 @@ export const CONFIG_DIRECTORY = getDirectory('config');
 export const LINK_REGISTRY_DIRECTORY = path.join(CONFIG_DIRECTORY, 'link');
 export const GLOBAL_MODULE_DIRECTORY = path.join(CONFIG_DIRECTORY, 'global');
 
+export const POSIX_GLOBAL_PREFIX = '/usr/local';
```

Comment:
No worries, the whole global-prefix has been a quite long conversation indeed.

Maybe the website docs can be updated with how we get the prefix so that regular users find the info easily, and why we don't mimic NPM.